server/containers/rpmrepo/Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM centos:7
+FROM centos:8
 
 ENV REPO_LOCAL_DIR=/var/tmp/repo/local/
 ENV REPO_INCOMING_DIR=/var/spool/repo/incoming/
 
 RUN yum install -y \
-        https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+        https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
     && yum install -y \
-        createrepo openssh-server bash incron sudo gnupg rpm-sign rsync \
-        yum-utils gcc python3 python3-devel libffi-devel \
-    && pip3 install gsutil \
+        createrepo openssh-server bash gnupg rpm-sign wget \
```

Comment:
I used wget because curl didn't follow GitHub redirects and downloaded the HTML redirect body instead.

Reply:
`curl -L`?